frontend/test/metabase/scenarios/dashboard/chained-filters.cy.spec.js

```diff
@@ -0,0 +1,83 @@
+import { signIn, restore, popover } from "__support__/cypress";
+
+describe("scenarios > dashboard > chained filter", () => {
+  before(restore);
+  beforeEach(signIn);
+
+  it("limit search options based on linked filter", () => {
+    cy.visit("/dashboard/1");
+    // start editing
+    cy.get(".Icon-pencil").click();
+
+    // add a state filter
+    cy.get(".Icon-filter").click();
+    popover().within(() => {
+      cy.findByText("Location").click();
+      cy.findByText("State").click();
+    });
+
+    // connect that to people.state
+    cy.findByText("Column to filter on")
+      .parent()
+      .within(() => {
+        cy.findByText("Select…").click();
+      });
+    popover().within(() => {
+      cy.findByText("State").click();
+    });
+
+    // open the linked filters tab, and click the click to add a City filter
+    cy.findByText("Linked filters").click();
+    cy.findByText("add another dashboard filter").click();
+    popover().within(() => {
+      cy.findByText("Location").click();
+      cy.findByText("City").click();
+    });
+
+    // connect that to person.city
+    cy.findByText("Column to filter on")
+      .parent()
+      .within(() => {
+        cy.findByText("Select…").click();
+      });
+    popover().within(() => {
+      cy.findByText("City").click();
+    });
+
+    // Link city to state
+    cy.findByText("Limit this filter's choices")
+      .parent()
+      .within(() => {
+        // turn on the toggle
+        cy.findByText("State")
+          .parent()
+          .within(() => {
+            cy.get("a").click();
+          });
```

Comment:
I saw this in some other test as well and wanted to ask you - is it possible that we update Toggle component to have one "clean" CSS class of `.Toggle`?

This is from a test: `Toggle__toggle___2lChr no-decoration Toggle__selected___3MRMF`. I see two "problems" here:
1. cannot get a class without "styled component" addition
2. (less importantly) if we want to follow BEM, then "selected" should be a modifier, thus `.Toggle--selected`

Hope you don't mind the comment, as it is not directly related to your PR?